drag_and_drop_v2/public/js/drag_and_drop_edit.js

```diff
@@ -70,9 +70,11 @@ function DragAndDropEditBlock(runtime, element, params) {
                         }
 
                         // Set the target image and bind its event handler:
-                        $('.target-image-form input', element).val(_fn.data.targetImg);
+                        $('.target-image-form input.url-input', element).val(_fn.data.targetImg);
+                        $('.target-image-form input.description-input', element).val(_fn.data.targetImgDescription);
                         _fn.build.$el.targetImage.load(_fn.build.form.zone.imageLoaded);
                         _fn.build.$el.targetImage.attr('src', params.target_img_expanded_url);
+                        _fn.build.$el.targetImage.attr('alt', params.target_img_description);
```

Comment:
I asked this earlier obliquely, but is there any difference between `_fn.data.targetImgDescription` and `params.target_img_description`?